### ISSUE TITLE
Problems with releases 2.2 and 2.1.2 on the tC releases page

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#7123]
https://github.com/unfoldingWord/translationCore/releases
Release 2.2 seems to have disappeared from this page
Downloading the 2.1.2 Win64 release downloads file translationCore-win-x64-2.1.2-44c92e9.setup.exe, but when the setup is run, 3.0.0 (74b79f2) gets installed.

**Comment by PhotoNomad0** [comment]
@cckozie I don't think we ever released a v2.2.0 - I think we changed the number to v3.0.0 before release.

**Comment by PhotoNomad0** [comment]
@cckozie ALso downloading and installing 2.1.2 on Windows worked fine for me.
